code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -229,9 +232,16 @@ Defined in conflicts.dm of the #defines folder.
 	loc = get_turf(G)
 
 
+/obj/item/attachable/proc/on_burst_amount_change(obj/item/weapon/gun/shoota, mob/user)
```

Comment:
I'd use G here

Reply:
I can use a different variable name to be less memey, if you have suggestions, but something more descriptive than `G` and in lower case would be appreciated.

Reply:
I like G there is nothing wrong with it, personally I'm a fan of this one letter schemes unless there's 2 very similar objects involved.